src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write code here
+const items = document.querySelectorAll('li');
+const tree = document.querySelector('.tree');
+
+for (const item of items) {
+  const span = document.createElement('span');
+
+  item.prepend(span);
+  span.append(span.nextSibling);
```

Comment:
The line `span.append(span.nextSibling);` is intended to move the existing text node of the list item into the newly created span. However, if the list item has no text node or if the next sibling is not a text node, this may not work as expected. Consider checking if `span.nextSibling` is a text node before appending it.